lib/internal/bootstrap_node.js

```diff
@@ -39,13 +39,13 @@
     _process.setup_cpuUsage();
     _process.setupMemoryUsage();
     _process.setupConfig(NativeModule._source);
-    NativeModule.require('internal/process/warning').setup();
-    NativeModule.require('internal/process/next_tick').setup();
-    NativeModule.require('internal/process/stdio').setup();
+    NativeModule.require('internal/process/warning')();
+    NativeModule.require('internal/process/next_tick')();
+    NativeModule.require('internal/process/stdio')();
     _process.setupKillAndExit();
     _process.setupSignalHandlers();
     if (global.__coverage__)
-      NativeModule.require('internal/process/write-coverage').setup();
+      NativeModule.require('internal/process/write-coverage')();
```

Comment:
Note: I used this naming convention for consistency with the other setup functions.

IMO, it would be better to preserve it for clarity.